cypress/integration/participantStatus.spec.js

```diff
@@ -9,21 +29,26 @@ describe('Participants status suite', () => {
     cy.contains('button', 'View My Candidates').click();
   };
 
-  const interviewParticipantForm = (siteId) => {
+  const interviewParticipantForm = () => {
     // let it select today as the date
-    cy.get('input[name=ContactedDate]').clear().type('2022/01/13');
+    // Ensure dates selected are within the phase range of Sacred Macaw
+    cy.get('input[name=ContactedDate]').clear().type('2023/01/13');
     cy.contains('button', 'Submit').click();
   };
 
   const hireParticipantForm = (siteId) => {
     // both dates
-    cy.get('input[name=DateHired]').clear().type('2022/01/14');
-    cy.get('input[name=StartDate]').clear().type('2022/01/15');
+    // Ensure dates selected are within the phase range of Sacred Macaw
+    cy.get('input[name=DateHired]').clear().type('2023/01/14');
+    cy.get('input[name=StartDate]').clear().type('2023/01/15');
     // select site
     cy.get('#mui-component-select-site').click();
     cy.get(`li[data-value='${siteId}']`).click();
     // acknowledge participant accepted offer in writing
     cy.get('input[name="acknowledge"]').click();
+    //  expect alert with allocations/remainingHires to exist
+    cy.wait(1000);
+    cy.get('.MuiAlert-message').contains(`This site has 100 allocations assigned`);
```

Comment:
Any reason for this to be a template?

```suggestion
    cy.get('.MuiAlert-message').contains('This site has 100 allocations assigned');
```

Reply:
Unintentional result of a copy/paste situation. I'll fix it!